ambry-api/src/main/java/com.github.ambry/config/RouterConfig.java

```diff
@@ -342,6 +342,13 @@
   @Default("600")
   public final long routerOperationTrackerHistogramDumpPeriod;
 
+  /**
+   * The max number of chunks per GetBlobOperation or PutOperation that may be buffered in memory.
+   */
+  @Config("router.max.in.mem.chunks")
+  @Default("4")
```

Comment:
How did we arrive at this default value?

Reply:
This came from the constant that was previously used, `NonBlockingRouter.MAX_IN_MEM_CHUNKS`